### PR TITLE
Orb removal

### PR DESCRIPTION
**Pull request by KyleTryon**
Update FAQ to include orb removal.

### CODE REVIEW

File: jekyll/_cci2/orb-author-faq.md [hunk 1]
```diff
@@ -29,6 +29,18 @@ Error: Unable to find organization YOUR_ORG_NAME of vcs-type GITHUB: Must have m
 
 Read more in the [Orb CLI Permissions Matrix]({{site.baseurl}}/2.0/orb-author-intro/#permissions-matrix).
 
+## Deleting Orbs
```
Comment (teka23) [inline jekyll/_cci2/orb-author-faq.md:32]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 23. You can fix by inserting `{: #deleting-orbs }` to the next line

File: jekyll/_cci2/orb-author-faq.md [hunk 2]
```diff
@@ -94,7 +106,7 @@ steps:
     name: Install Homebrew (for Linux)
 ```
 
-## Command vs job
+## Command vs Job
```
Comment (teka23) [inline jekyll/_cci2/orb-author-faq.md:109]: JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 100. You can fix by inserting `{: #command-vs-job }` to the next line

File: jekyll/_cci2/orb-author-faq.md (hunk 1 above)
Comment (KyleTryon) [reply jekyll/_cci2/orb-author-faq.md:33]: Added to all headers 👍 thank you. LMK how this looks.

File: jekyll/_cci2/orb-author-faq.md (hunk 2 above)
Comment (KyleTryon) [reply jekyll/_cci2/orb-author-faq.md:114]: Added

**Review by teesloane — APPROVED** [review]

**Pull request merged**